owl/SOMA-OBJ.owl

```diff
@@ -790,6 +828,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasForceValue -->
+
+    <owl:DatatypeProperty rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#hasForceValue">
+        <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#hasRegionDataValue"/>
+        <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Region"/>
```

Comment:
Should this property have any region as domain?

Reply:
I added it intentionally, yes. Because I didn't want to exclude things that are not modeled. So this net force attribute is in my opinion not the only region that could have a force value. That's why I left it with the general domain axiom

Reply:
Would Color have a force, or Shape? I don't think it will make sense, except perhaps as metaphor but we don't care about those, to have force values for most regions. Would it then make sense to have a ForceRegion class or similar, with NetForce as a subclass?

Reply:
Is every force region an attribute then?

Reply:
What's an attribute?

Reply:
`PhysicalAttribute` in DUL is a "Physical value of a physical object, e.g. density, color, etc.".
I have defined NetForce as subclass of PhysicalAttribute

Reply:
Then I would basically define `ForceAttribute` and `NetForce` as a sub-attribute. Then I could add an axiom about hasForceValue only giving a value to force attributes

Reply:
That seems like a plan, let's have it so.